test/utils.go

```diff
@@ -110,11 +110,11 @@ func generate(t *testing.T, amount uint64, address string) string {
 	return blockHash
 }
 
-func execCommandGenerate(t *testing.T, amount uint64, address string) string {
+func execCommandGenerate(t *testing.T, amount uint64) string {
```

Comment:
Let's rename all functions with prefix `exec` as we're now the `go-bitcoin` package functions